src/convertToObject.js

```diff
@@ -5,8 +5,25 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+function convertToObject(stylesString) {
+  const stylesObject = {};
+
+  const stylesArray = stylesString
+    .split(';')
+    .filter((style) => style.trim() !== '');
+
+  stylesArray.forEach((style) => {
+    const [property, ...valueParts] = style.split(':');
+    const value = valueParts.join(':').trim();
+
+    if (property && value) {
+      const cleanedValue = value.replace(/\n\s+/g, (match) => match);
+
+      stylesObject[property.trim()] = cleanedValue;
```

Comment:
[CODE KNOWLEDGE] - The replacement function in line 20 is unnecessary since `.trim()` in line 17 already handles the trimming of whitespace. The use of the replacement function does not contribute to the logic and can be removed.